ros2doctor/test/test_report.py

```diff
@@ -130,11 +144,6 @@ def test_check(self) -> None:
 
     @launch_testing.markers.retry_on_failure(times=5, delay=1)
     def test_report(self) -> None:
-        # TODO(@fujitatomoya): rmw_zenoh_cpp is instable to find the endpoints, it does not
-        # matter if DaemonNode or DirectNode is used. For now, skip the test for rmw_zenoh_cpp.
-        if self.rmw_implementation == 'rmw_zenoh_cpp':
-            raise unittest.SkipTest()
-
```

Comment:
This is another fix to support zenoh here. It now uses rmw isolated environment for the test, that will call rmw_zenoh specific setup for the isolation.